app/presenters/post_presenter.rb

```diff
@@ -47,6 +47,10 @@ def page_title
     post_page_title @post
   end
 
+  def poll_participation_answer_id
```

Comment:
The only reason you made this method public is because you need it in the tests. But you shouldn't change code only for tests. Can you please move this method down in the private scope (for example where `already_participated_in_poll` was).